### PR TITLE
Release controller 0.10.3 and helm chart 0.12.3

### PR DESCRIPTION
**Pull request by Abdiramen**
Release the Gateway 🦑

### CODE REVIEW

**Comment by Abdiramen** [comment]
* **#350** <a href="https://app.graphite.dev/github/pr/ngrok/kubernetes-ingress-controller/350?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Abdiramen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
```
Comment (nikolay-ngrok) [inline CHANGELOG.md:9]: update the release number here?

Comment (Abdiramen) [reply CHANGELOG.md:9]: updated!

**Review by nikolay-ngrok — APPROVED** [review]
👍

File: CHANGELOG.md (hunk 1 above)
Comment (e88301er) [reply CHANGELOG.md:9]: ngrok

**Pull request merged**